src/pages/productList/ProductList.tsx

```diff
@@ -0,0 +1,3 @@
+export default function ProductList() {
+  return <div></div>;
+}
```

Comment:
저희 선언방식은 다 함수형컴포넌트 (rafce tab 후 import react 삭제하기) 
```suggestion
const Home = () => {
  return (
    <div>
      <h1>Home Page</h1>
    </div>
  );
};

export default Home;
```

로 작성해주세요!

Reply:
넵!